wasm/src/lib.rs

```diff
@@ -23,11 +23,13 @@ pub use iroha_data_model as data_model;
 pub use iroha_wasm_derive::entrypoint;
 pub use parity_scale_codec::{DecodeAll, Encode};
 
-pub mod debug;
+mod debug;
```

Comment:
why `debug` became private?

Reply:
I didn't notice. I can either make `debug` module private or remove `pub use debug::*` statement. The latter seems like a better option. What do you say?

Reply:
let's remove `pub use debug::*`